code/modules/events/blizzard.dm

```diff
@@ -3,12 +3,12 @@ var/blizzard_cooldown = 5 MINUTES
 
 
 /datum/event/blizzard/can_start()
-	return 80 * istype(map.climate,/datum/climate/arctic)
```

Comment:
wouldn't it make more sense to search the entire list of climates and check if ANY is arctic?